### DISCUSSION TITLE
Can't upload file with single quotes

### DISCUSSION THREAD

**Discussion by twilson90** [Studio-42/elFinder#3346] [Q&A]
Simply get a generic error 'Upload error.'

The problem occurs in protected function upload($args) in elFinder.class.php.
The $chunk variable (containing the filename) is escaped with backslashes, and then when it checks the chunk on line 3186 it returns the error.

This is evident on my Windows testing machine and my remote Ubuntu 20.04 machine, both with PHP 7.4.16

So why are backslashes being added to the $chunk variable?

**Comment by nao-pon** [comment]
@hedgehog90 I've uploaded a file with single quotes on the demo site and it doesn't seem to be a problem.

Is something working in your server environment that escapes single quotes?

**Reply by twilson90** [reply]
It was WordPress and its stupid 'magic quoting' all the $_ variables, so I had to unmagic quote them at the top of my elfinderConnector.php
